### PR TITLE
fix(docs): add ignored-branches documentation

### PR DESCRIPTION
**Pull request by neha-sri**

### CODE REVIEW

**Comment by neha-sri** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11**  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: guides/graphite-cli/cli-command-reference.md [hunk 1]
```diff
@@ -205,7 +219,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main or suggests to resubmit a branch whose PR base has changed.
+    * Don't prompt confirmation when Graphite suggest deleting a branch already-merged into main or suggests to resubmit a branch whose PR base has changed.
```
Comment (xiulung) [inline guides/graphite-cli/cli-command-reference.md]: Why is "already-merged" hyphenated?

Nit: "suggest" should be "suggests"

What is the default behavior here? What is the default action if no confirmation is prompted?

The wording is confusing to me here because suggestions implies "We think you should do this thing. Do you want to do it? Yes/No" and confirmation implies "You want to do this thing. Are you sure you want to do it? Yes/No"

File: guides/graphite-cli/cli-command-reference.md [hunk 2]
```diff
@@ -224,7 +238,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main.
+    * Don't prompt confirmation when Graphite suggests deleting a branch already-merged into main.
```
Comment (xiulung) [inline guides/graphite-cli/cli-command-reference.md]: Same question re:hyphenation of "already-merged"

Nit: let's be wordier in favor of readability by adding "which is"

"Don't prompt confirmation when Graphite suggests deleting a branch which is already merged into main."

File: guides/graphite-cli/cli-command-reference.md [hunk 3]
```diff
@@ -205,11 +219,11 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main or suggests to resubmit a branch whose PR base has changed.
+    * Suppress prompts when Graphite makes suggestions for deleting a branch which is already merged into main or to resubmit a branch whose PR base has changed.
```
Comment (xiulung) [inline guides/graphite-cli/cli-command-reference.md]: This part still sounds like Graphite is still making suggestions, but they’re simply being suppressed.

If what we’re saying is Graphite will not make the suggestion in the first place. Then I would suggest changing this to follow the same pattern as the other lines:

“Skip the steps where Graphite checks whether feature branches have been merged into trunk, and suggests resubmitting branches whose PR bases differ locally from remote."

File: guides/graphite-cli/cli-command-reference.md [hunk 4]
```diff
@@ -224,7 +238,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main.
+    * Suppress prompts when Graphite makes suggestions for deleting a branch which is already merged into main.
```
Comment (xiulung) [inline guides/graphite-cli/cli-command-reference.md]: Suggest rewording to fit the "Skip the step..." pattern

File: guides/graphite-cli/cli-command-reference.md (hunk 3 above)
Comment (neha-sri) [reply guides/graphite-cli/cli-command-reference.md]: So we don't skip the check, we just suppress the prompt and take default action.

Comment (xiulung) [reply guides/graphite-cli/cli-command-reference.md]: Ah I see, what is the default action? Is it a user setting?

Comment (neha-sri) [reply guides/graphite-cli/cli-command-reference.md]: Sometimes. If they supply additional options like `--draft` then we use that, else we choose the conservative option based on the default choices indicated in the `--help` menu.

Comment (xiulung) [reply guides/graphite-cli/cli-command-reference.md]: Okay, then maybe something like

"Skip prompting to delete feature branches that have been merged into trunk, and suggesting resubmission of branches whose PR bases differ locally from remote. Instead, Graphite will take the default actions ..."

Followed by explaining what the default action is or how it's set

File: guides/graphite-cli/cli-command-reference.md [hunk 5]
```diff
@@ -224,7 +238,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main.
+    * Skip prompting to delete feature branches that have been merged into trunk. (may include deleting already merged branches and setting branch parents).
```
Comment (xiulung) [inline guides/graphite-cli/cli-command-reference.md]: Let's add the note about the default action instead. Otherwise LGTM

**Review by xiulung — APPROVED** [review]

File: guides/graphite-cli/cli-command-reference.md (hunk 5 above)
Comment (neha-sri) [reply guides/graphite-cli/cli-command-reference.md]: These are the default actions? Delete merged branches and rebase. This is also the language used in `--help` of `repo sync`
